src/romaine/parser/__init__.py

```diff
@@ -0,0 +1,34 @@
+from romaine.parser import (
+    simple,
+    multiline,
+    step,
+    section,
+    feature,
+    )
```

Comment:
Yeah, I was trying to do all of these in the style where you do closing parens on the same level of indentation as the thing they're closing. I'm not sure I was consistent on that though. Inherited it from a friend and I actually think it looks nicer but it's so infrequently done that I'll sort it now.

Un(?)fortunately, flake8 doesn't complain about it, so I'll have to search for it by hand.